src/CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ add_library(PYSTON_OBJECTS OBJECT ${OPTIONAL_SRCS}
 		runtime/util.cpp
 		)
 
-add_dependencies(PYSTON_OBJECTS libunwind_patched libunwind pypa ${LLVM_LIBS})
+add_dependencies(PYSTON_OBJECTS FROM_CPYTHON libunwind_patched libunwind pypa ${LLVM_LIBS})
```

Comment:
Does it work to make it depend on `copy_stdlib` instead?  (I'm assuming this additional dependency comes from switching the include dir to the copy in the build directory.)